core/src/main/kotlin/com/alecstrong/sql/psi/core/psi/mixins/SelectStmtMixin.kt

```diff
@@ -55,7 +53,7 @@ internal abstract class SelectStmtMixin(
   }
 
   override fun queryAvailable(child: PsiElement): Collection<QueryResult> {
-    if (child in exprList) {
+    if (child in exprList + (groupBy?.exprList ?: emptyList())) {
```

Comment:
i think this can be more efficient without the allocation by doing `child in exprList || child in )groupBy?.exprList ?: emptyList())`

Reply:
I don't know if our performance is really important when using PSI :D